Dockerfile-server

```diff
@@ -19,4 +15,4 @@ RUN apk add --update tzdata
 ENV TZ=Asia/Shanghai
 RUN rm -rf /var/cache/apk/*
 RUN chmod +x server
-ENTRYPOINT ./server -db $mongodb -es $elasticsearch
+ENTRYPOINT for i in $(seq 1 5);do echo "Wait for ElasticSearch Start...$i"; curl -s http://$IDS_ELASTICSEARCH> /dev/null ; if [ $? = 0 ];then break; fi; sleep 1; if [ $i == 5 ];then echo "Too much retry, exit."; exit 1; fi; done; ./server -db $IDS_MONGODB -es $IDS_ELASTICSEARCH
```

Comment:
🤔I think to put all of these into a shell script and make that as the entry point would be clear.

Reply:
okay